src/Test.Utilities.Sarif/FileDiffingUnitTests.cs

```diff
@@ -27,19 +27,29 @@
     {
         public static string GetTestDirectory(string subdirectory = "")
         {
-            return Path.GetFullPath(Path.Combine(@$"TestData\", subdirectory));
+            return Path.GetFullPath(Path.Combine("TestData", subdirectory));
         }
 
         public static string GetProductDirectory()
         {
             string path = typeof(FileDiffingUnitTests).Assembly.Location;
-            path = GitHelper.Default.GetTopLevel(path);
```

Comment:
>GetTopLevel [](http://example.com/codeflow?start=37&length=11)

File an issue to revert this change, by updating GitHelper.Default.GetTopLevel go look for a '.git' directory on the Linux platform.

For extra credit, do this for Windows as well if git.exe does not resolve. :)

Reply:
created
[https://github.com/microsoft/sarif-sdk/issues/2450](https://github.com/microsoft/sarif-sdk/issues/2450)